src/math/utils.c

```diff
@@ -255,21 +256,21 @@ int igraph_is_neginf(double x) {
 
 /**
  * \function igraph_almost_equals
- * Compare two double-precision floats with a tolerance
+ * \brief Compare two double-precision floats with a tolerance.
  *
  * Determines whether two double-precision floats are "almost equal"
  * to each other with a given level of tolerance on the relative error.
  *
- * \param  a  the first float
- * \param  b  the second float
- * \param  eps  the level of tolerance on the relative error. The relative
- *         error is defined as \c "abs(a-b) / (abs(a) + abs(b))". The
+ * \param  a  The first float.
+ * \param  b  The second float.
+ * \param  eps  The level of tolerance on the relative error. The relative
+ *         error is defined as <code>abs(a-b) / (abs(a) + abs(b))</code>. The
  *         two numbers are considered equal if this is less than \c eps.
  *
- * \return nonzero if the two floats are nearly equal to each other within
- *         the given level of tolerance, zero otherwise
+ * \return Nonzero if the two floats are nearly equal to each other within
+ *         the given level of tolerance, zero otherwise.
```

Comment:
Let's say true and false here. The return type is not a Boolean (it used to be an integer).